tests/compiler/test_basics.py

```diff
@@ -636,7 +637,8 @@ def get_one() -> "int":
     )
     with warnings.catch_warnings(record=True) as w:
         compiler.do_compile()
-        assert len(w) == 1
-        warning = w[0]
+        assert len(w)
+        # first warning is about the use of V1 modules that are deprecated.
+        warning = w[1]
```

Comment:
Same here. The warning we expect should not necessarily be present at index 1. It could be present at any other position.